owners/index.js

```diff
@@ -34,7 +34,7 @@ module.exports = app => {
   const localRepo = new LocalRepository(process.env.GITHUB_REPO_DIR);
   const ownersBot = new OwnersBot(localRepo);
   const sharedGithub = new GitHub(
-    new Octokit({auth: `token ${GITHUB_ACCESS_TOKEN}`}),
+    new Octokit({auth: GITHUB_ACCESS_TOKEN}),
```

Comment:
😬

Reply:
Octokit docs are terrible and inconsistent :unamused: